lib/lumberjack/server.rb

```diff
@@ -56,7 +67,7 @@ def run(&block)
 
     def accept(&block)
       begin
-        fd = @ssl_server.accept
+        fd = @server.accept
       rescue EOFError, OpenSSL::SSL::SSLError, IOError
         # ssl handshake or other accept-related failure.
         # TODO(sissel): Make it possible to log this.
```

Comment:
We might need to adjust here what errors we are trapping and what we do with them depending of the connection mode. See my comment here. https://github.com/elastic/ruby-lumberjack/pull/14/files#r39822455